src/test/java/com/google/ehub/servlets/LoginServletTest.java

```diff
@@ -0,0 +1,105 @@
+package com.google.ehub.servlets;
+
+import static org.junit.Assert.assertFalse; 
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.*; 
+
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.appengine.api.users.User;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalUserServiceTestConfig;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+ 
+@RunWith(JUnit4.class)
+public class LoginServletTest {
+  private static final String JSON_CONTENT_TYPE = "application/json";
+  private static  String LOGOUT_URL = "/logout";
+  private static  String LOGIN_URL = "/login";
+  private static final String userId = "12345";
+  private static final String email = "abc@gmail.com";
+  private static final String authDomain = "gmail.com";
+  private static final User user= new User(email, authDomain , userId);
```

Comment:
These are constants, so they should be CONSTANT_CASE: all uppercase letters, with each word separated from the next by a single underscore. 

e.g. `userId` -> `USER_ID`

Reply:
These lines were removed